src/lucky/tags/select_helpers.cr

```diff
@@ -9,7 +9,7 @@ module Lucky::SelectHelpers
 
   def options_for_select(field : LuckyRecord::AllowedField, select_options : Array(Tuple(String, SelectOption)), **html_options)
     select_options.each do |option_name, option_value|
-      attributes = { "value" => option_value.to_s }
+      attributes = {"value" => option_value.to_s}
```

Comment:
Is it convention to not use padded hashes?

Reply:
Yep! All the style changes are done by `crystal tool format` automatically. So yeah it is the Crystal convention